test/e2e/run-tests

```diff
@@ -47,13 +45,10 @@ ALL_INSTANCE_TYPES=$SUPPORTED_INSTANCE_TYPES,$UNSUPPORTED_INSTANCE_TYPE_AMI,$UNS
 
 function init_test_resources() {
   # create resources used for e2e tests
-  AWS_DEFAULT_REGION=$DEFAULT_REGION
```

Comment:
rather than specifying `--region` on every command, you could `export AWS_REGION=$DEFAULT_REGION` here and unset it at the bottom of the function.  OR if the region should always be the same (which I think it should now that the non-default test has been removed), you could just specify it at the top of the file and not worry about setting and unsetting in the functions.